style.css

```diff
@@ -0,0 +1,80 @@
+body>section{
+    background-color: lightgrey;
+    max-width: 625px;
+    max-height: 800px;
+    margin-left: auto;
+    margin-right: auto;
+}
+.image{
+    position: relative;
```

Comment:
I see that your solution to not using padding or margin was to use position property. While it works in a small example like here, it should be used sparingly. The point of this exercise was to make you explore the commonly overlooked part of the box-model - border. Use the `border` property instead of position relative. You can set a size for it and then make it transparent!